src/Components/StudyReviews/PeerToPeerReview/Review/index.tsx

```diff
@@ -0,0 +1,108 @@
+import { ReviewType } from '@/Types/review';
+import styled from 'styled-components';
+import { Profile } from '@/Assets';
+import { ReviewScore } from '@/Components/ReviewScore';
+
+interface ReviewProps extends Partial<ReviewType> {
```

Comment:
```suggestion
export interface ReviewProps extends Partial<ReviewType> {
```
export로 내보내주면 해당 타입을 재사용하기 쉬워질 것 같습니다!